### PR TITLE
Hover banner resizing

### PR DESCRIPTION
**Pull request by Kalekdan**
## What does this do?

Replaces the border property on hovering on the 'roll new town' button with an inset shadow. The border property was causing the span to increase by a pixel on either side, so was resulting in jittering.

## How was this tested? Did you test the changes in the compiled `.html` file?

Built locally and verified the changes.

## Is there a [GitHub Issue](https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/issues?q=is%3Aopen+is%3Aissue) that this is resolving?

#762

### CODE REVIEW

File: src/Settings/CSS/stylesheet.css [hunk 1]
```diff
@@ -282,11 +282,7 @@ html.two-columns #illustration-buffer {
 
 .restart:hover {
   background-color: var(--background-secondary);
```
Comment (ryceg) [inline src/Settings/CSS/stylesheet.css:284]: So, I tested this on dark mode, and it's not great. Might wanna remove this.

File: src/Settings/CSS/stylesheet.css [hunk 2]
```diff
@@ -282,11 +282,7 @@ html.two-columns #illustration-buffer {
...
-  /* border-color: var(--text-normal); */
-  border-left: 1px solid var(--text-normal);
-  border-right: 1px solid var(--text-muted);
-  border-top: 1px solid var(--text-muted);
-  border-bottom: 1px solid var(--text-muted);
+  box-shadow: 0px 0px 0px 1px  var(--text-normal) inset;
```
Comment (ryceg) [inline src/Settings/CSS/stylesheet.css:285]: Good stuff! Maybe also add a transition?

**Review by ryceg — CHANGES_REQUESTED** [review]

File: src/Settings/CSS/stylesheet.css (hunk 1 above)
Comment (Kalekdan) [reply src/Settings/CSS/stylesheet.css:284]: Do you mean to remove the background-color? This has not been changed and is as it exists in the live version. I can remove if desired just wanted to check first!

**Comment by Kalekdan** [comment]
Changes made - ready to be merged and closed if approved :)

**Review by ryceg — APPROVED** [review]

**Comment by ryceg** [comment]
Closes #762 Thanks @Kalekdan!

**Pull request merged**